pubsub/spec/sample_spec.rb

```diff
@@ -0,0 +1,146 @@
+# Copyright 2015 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "../sample"
+require "spec_helper"
+
+describe "Pub/Sub sample" do
+  TOPIC_NAME = "my-topic".freeze
+  SUB_NAME = "my-subscription".freeze
+
+  before(:all) do
+    @gcloud = Gcloud.new ENV["GCLOUD_PROJECT"]
+    @pubsub = @gcloud.pubsub
+  end
+
+  before(:each) do
+    topic = @pubsub.topic TOPIC_NAME
+    topic.delete if topic
+    sub = @pubsub.subscription SUB_NAME
+    sub.delete if sub
+
+    allow(Gcloud).to receive(:new).with("my-gcp-project-id").and_return(@gcloud)
+  end
+
+  it "creates topic" do
+    expect(@pubsub.topic(TOPIC_NAME)).to be nil
+
+    expect { create_topic }.to output(/#{TOPIC_NAME}/).to_stdout
+
+    topic = @pubsub.topic TOPIC_NAME
```

Comment:
This can simply be `expect(@pubsub.topic TOPIC_NAME).to be`

The follow-ups aren't really necessary here or in any of the tests.